tests/functional/test_lint_imports.py

```diff
@@ -15,6 +15,34 @@
         (testpackage_directory, ".brokencontract.ini", cli.EXIT_STATUS_ERROR),
         (testpackage_directory, ".malformedcontract.ini", cli.EXIT_STATUS_ERROR),
         (testpackage_directory, ".customkeptcontract.ini", cli.EXIT_STATUS_SUCCESS),
+        (testpackage_directory, ".externalbrokencontract.ini", cli.EXIT_STATUS_ERROR),
```

Comment:
I've reordered the tests here so that the TOML ones were together. I also added a couple of tests for the TOML version of `setup.cfg` so we can make sure the wildcards are ok in that format.